### PR TITLE
Update serviceprincipalriskdetection.md

### PR DESCRIPTION
**Pull request by CrysCyber**
I am adding a new Enum to reflect a new detection

### CODE REVIEW

**Comment by Lauragra** [comment]
@CrysCyber , is this PR ready for content review? You will need to include a changelog entry for the addition of the new enum member. For details, see: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog

**Comment by cloudhandler** [comment]
@CrysCyber / @ebasseri to confirm- is it a new enum member or a new string? riskdetectiontype is a string for user riskdetections API?

File: api-reference/beta/resources/serviceprincipalriskdetection.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ For more information about risk events, see [Azure Active Directory Identity Pro
...
-|riskEventType|String|The type of risk event detected. The possible values are:  `investigationsThreatIntelligence`, `generic`, `adminConfirmedServicePrincipalCompromised`, `suspiciousSignins`, `leakedCredentials`, `unknownFutureValue`. Supports `$filter` (`eq`).|
+|riskEventType|String|The type of risk event detected. The possible values are:  `investigationsThreatIntelligence`, `generic`, `adminConfirmedServicePrincipalCompromised`, `suspiciousSignins`, `leakedCredentials`, `unknownFutureValue`, `anomalousServicePrincipalActivity`. Supports `$filter` (`eq`).|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/serviceprincipalriskdetection.md:48]: Is this a brand new member? If so, please add a changelog entry as Laura suggested and also document the new member where the enum was defined: probably in the enums.md file.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Please add the new member to the enum definition table and include a changelog entry.

**Pull request closed without merging**